feedhenry-raincatcher/raincatcher-core
# [RAINCATCH-1151] Add hashed index to id field for generated collections

## Motivation
Add indexing on custom `id` field when generating demo data.

## Description
Add extra logging and [hashed](https://docs.mongodb.com/manual/core/index-hashed/) indexing on the id field.

## Progress
- [x] Add index creation before seed data

## Additional Notes
To verify:

- Delete the `raincatcher` mongodb database
- Start the `demo-server` to trigger the seeding of the demo data
- Verify the new index by running `db.collection('workflows').getIndexes()` and `db.collection('workorders').getIndexes()` on a mongo console
